aep/general/0135/aep.md.j2

```diff
@@ -4,15 +4,45 @@ In REST APIs, it is customary to make a `DELETE` request to a resource's URI
 (for example, `/v1/publishers/{publisher}/books/{book}`) in order to delete
 that resource.
 
-Resource-oriented design (AIP-121) honors this pattern through the `Delete`
-method. These RPCs accept the URI representing that resource and usually return
+Resource-oriented design ([AEP-121][aep-121]) honors this pattern through the `Delete`
+method. These operations accept the URI representing that resource and usually return
 an empty response.
 
 ## Guidance
 
 APIs **should** generally provide a delete method for resources unless it is
 not valuable for users to do so.
 
+The Delete method **should** succeed if and only if a resource was present and
+was successfully deleted. If the resource did not exist, the method **should**
+send a `404 Not found` (`NOT_FOUND`) error.
+
+### Requests
+
+{% tab oas -%}
```

Comment:
I don't have an opinion on _which_ comes first, but we should probably try to be consistent about whether the proto or OAS tab comes first.

I think it's typically the other way around.  (But if it's already inconsistent then we should fix it in a separate PR.)

Reply:
Agree to fix in a separate PR.